changelog.dd

```diff
@@ -73,6 +82,84 @@ assert(m.front[1] == "12");
 -------
 )
 
+$(LI $(LNAME2 optimization, `findLocalMin` was added to `std.numeric`.)
+    $(P $(REF findLocalMin, std,numeric) finds a real minimum of a real function `f(x)` via bracketing.)
+-------
+import std.numeric, std.math;
+
+auto ret = findLocalMin((double x) => (x-4)^^2, -1e7, 1e7);
+
+assert(ret.x.approxEqual(4.0));
+assert(ret.y.approxEqual(0.0));
+-------
+)
+
+$(LI $(LNAME2 slice_ptr, `ptr` property and public constructor were added to
+    `std.experimental.ndslice.slice.Slice`.)
+    $(P `ptr` property allows to access `Slice`'s
+        underlaying pointer or range. It is suggested to read about
+        $(REF Slice, std,experimental,ndslice,slice)
+        internal binary representation before using the property.
+        `ptr` property is used in $(LINK2 https://github.com/libmir/mir, Mir)
+        for $(LINK2 http://docs.mir.dlang.io/latest/mir_sparse_package.html, sparse tensors).
+        `ndslice` developer mirror in Mir will be removed as
+        soon as LDC (LLVM D compiler) supports DMDFE 2.072.
```

Comment:
> DMDFE

I've never seen this abbreviated like this before, and it's kind of "[inside baseball](https://en.wikipedia.org/wiki/Inside_baseball_(metaphor))". I think something like "as soon as LDC (LLVM D compiler) supports D version 2.072." would be clearer.